packages/flutter_tools/lib/src/macos/application_package.dart

```diff
@@ -137,7 +137,7 @@ class PrebuiltMacOSApp extends MacOSApp {
 }
 
 class BuildableMacOSApp extends MacOSApp {
-  BuildableMacOSApp(this.project);
+  BuildableMacOSApp(this.project) : super(projectBundleId: '');
```

Comment:
This is not correct either; the bundle identifier for a macOS application is not an empty string. The logic for BuildableMacOSApp should look more like BuildableIOSApp, and parse the information out of the necessary project files.

Reply:
In particular, `macos/Runner/Configs/AppInfo.xcconfig` is where this information will be by default, but we should check the Xcode project first in case it's been overridden there by the developer.

Reply:
what should i do now？

Reply:
I'm not sure what your question is exactly; the next step if you want to proceed with the PR is to fix it as I described above.